change/@internal-react-components-cbda6e4f-8622-4f48-bcc9-42d5fe413676.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
This should have been `none` as the PR has no user-visible changes (nothing changes for Contoso using our library).